pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -103,6 +103,49 @@ func MonEndpoints(mons map[string]*cephclient.MonInfo, requireMsgr2 bool) []stri
 	return endpoints
 }
 
+// updateNetNamespaceFilePath modify the netNamespaceFilePath for all cluster IDs.
+// If holderEnabled is set to true. Otherwise, removes the netNamespaceFilePath value
+// for all the clusterIDs.
+func updateNetNamespaceFilePath(cc csiClusterConfig) {
+	var (
+		cephFSNetNamespaceFilePath string
+		rbdNetNamespaceFilePath    string
+		nfsNetNamespaceFilePath    string
+	)
+
+	if IsHolderEnabled() {
+		for _, centry := range cc {
+			if centry.CephFS.NetNamespaceFilePath != "" {
+				cephFSNetNamespaceFilePath = centry.CephFS.NetNamespaceFilePath
+			}
+			if centry.RBD.NetNamespaceFilePath != "" {
+				rbdNetNamespaceFilePath = centry.RBD.NetNamespaceFilePath
+			}
+			if centry.NFS.NetNamespaceFilePath != "" {
+				nfsNetNamespaceFilePath = centry.NFS.NetNamespaceFilePath
+			}
+		}
+
+		for i, centry := range cc {
```

Comment:
can we use `Namespace` in the configmap which is the ceph cluster namespace, if the namespace match then update the request NetNamespace path?